cfn-resources/cluster/mongodb-atlas-cluster.json

```diff
@@ -341,6 +341,10 @@
         "NONE"
       ]
     },
+    "GlobalClusterSelfManagedSharding": {
+      "description": "Flag that indicates if cluster uses Atlas-Managed Sharding (false) or Self-Managed Sharding (true).",
```

Comment:
as this is the only doc, I'll probably go for the long explanation (the one in the TF resource)